channels/microsoft365/accounts/register-app.rst

```diff
@@ -48,9 +48,21 @@ To get started, head over to `Microsoft’s Azure Portal <https://portal.azure.c
    use the following:
 
    Supported account types
-      **Accounts in any organizational directory
-      (Any Azure AD directory - Multitenant) 
-      and personal Microsoft accounts (e.g. Skype, Xbox)**.
+      Choose from one of the following options:
+
+      * Accounts in this organizational directory only 
+        (Default Directory only - Single tenant)
+      * Accounts in any organizational directory 
+        (Any Azure AD directory - Multitenant)
+      * Accounts in any organizational directory 
+        (Any Azure AD directory - Multitenant) 
+        and personal Microsoft accounts (e.g. Skype, Xbox)
+
+      .. tip::
+
+         The correct account type depends on your use case. 
+         If you want to use the authentication internal only, choose the first 
+         option. If you're unsure, use the "Help me choose..." link.
```

Comment:
IMO this is too verbose. I would prefer

```suggestion
      Choose the option that’s right for your organization
      (or click *Help me choose...* if you’re not sure).

      .. note:: 🙅 **The “Personal Microsoft accounts only” option is not supported.**
```

Reply:
Okay. I was aiming to have exakt settings written down in my "get em all mode". 
Yours is slicker and neater - I'll change it to a longer version in case it proves to be not enough. 💪